### DISCUSSION TITLE
Hotkey or Single Key Start/Stop Playback

### DISCUSSION THREAD

**Discussion by maldons** [AntonyCorbett/OnlyM#450] [Ideas]
I had an idea where a single key could be used to start playing the media files.  So in the order that the images are in OnlyM, pressing "1" would start that piece of media and pressing "1" again would stop the media. If "2" were pressed, the media in the second position would be played and down the line of numbers. All the numbers could then be used for each different media file.  Letters could be used for the 10th media piece, etc.  Space bar (or zero) could be a universal pause/resume hot key.  The media playlist could display the key that needs to be pressed for playback giving the necessary visual indicator for the operator.

I realize that OnlyM would need to be the focus application for this to work, but it would give some hot key combinations that could be used on the Keyboard, with AutoHotKey or with a Stream Deck for single push button media control. 

Thank you.

**Comment by ColinAAvey** [comment]
Some ideas to add to the above.

- Have the HotKeys configurable, so they don't clash with other software which may be being used. Some congregations use ManyCam for instance, which has many Gloabl HoyKeys.
- Maybe consider just having HotKeys for Play/Pause. Play Next Media and Play Previous Media instead of 1,2,....a.b...
We already duplicate any media which is re-used in a talk and prefix the name with 01,02,03, so the brother just have to work through the list in sequence rather than find a specific item.
If no item has been selected, then Play/Pause & Play Next can default to the first item in the list.

Just some ideas

**Comment by AntonyCorbett** [comment]
See also #46
